tests/integration/test_pytest_operator.py

```diff
@@ -53,6 +53,24 @@ async def test_0_shared_model_and_test_order(self, ops_test):
             "operator-framework",
         }
 
+    async def test_1_create_crash_dump(self, ops_test):
+        """Check if juju-crashdump was called."""
+        lib_path = Path(__file__).parent.parent.parent
+        user_crashdumps = set(lib_path.glob("juju-crashdump-*.tar.xz"))
```

Comment:
```Path(__file__).parent.parent.parent``` is the same directory as the main repository path.
Why is there a `juju-crashdump-*.tar.xz` here?  If you need a temporary directory guaranteed to be empty, try `ops_test.tmp_path`.

I think you should create your crashdumps there.

Reply:
You are right, since I do not use the main repository path I should change this. `ops_test.crash_dump_output = ops_test.tmp_path`

Actually is a `juju-crashdump-*.tar.xz` was not anywhere, the test was XFAILD, due missing juju-crashdump command. I didn't want it to depend on this [PR](https://github.com/charmed-kubernetes/actions-operator/pull/31).

Reply:
[Here](https://pastebin.ubuntu.com/p/gXzPfp5Qvr/) is output from my local run.